calico/felix/ipsets.py

```diff
@@ -317,8 +317,6 @@ def on_endpoint_update(self, endpoint_id, endpoint):
             for ip in old_ips:
                 self._remove_mapping(tag, profile_id, endpoint_id, ip)
 
-        _log.info("Endpoint update complete")
```

Comment:
If this log provided any value it probably would have been better to drop the severity rather than remove it, but there's no good reason for that to block this merge.